From d1555ffdd9b4fa6f3ceaa166bbfee0b3b9973ecf Mon Sep 17 00:00:00 2001
From: David Malcolm <dmalcolm@redhat.com>
Date: Wed, 1 Sep 2021 11:24:34 -0400
Subject: [PATCH 10/17] Don't verify exception handling in basic_filebuf::close
 in DTS [PR81256]

27_io/basic_filebuf/close/81256.cc was added upstream 2018-05-14 in
  PR libstdc++/81256 fix exception handling in basic_filebuf::close
    https://gcc.gnu.org/git/?p=gcc.git;a=commitdiff;h=e77497ab777d3dfa19224648d658220807ab7419

When run under DTS,
  std::basic_filebuf<wchar_t, std::char_traits<wchar_t> >::close
comes from the system libstdc++, and hence the test fails.
---
 libstdc++-v3/testsuite/27_io/basic_filebuf/close/81256.cc | 1 -
 1 file changed, 1 deletion(-)

--- a/libstdc++-v3/testsuite/27_io/basic_filebuf/close/wchar_t/81256.cc
+++ b/libstdc++-v3/testsuite/27_io/basic_filebuf/close/wchar_t/81256.cc
@@ -80,7 +80,6 @@ test01()
       caught = true;
     }
     VERIFY( conv.exceptions_thrown == 1 );
-    VERIFY( caught );
   }
   VERIFY( conv.exceptions_thrown == 1 );
 }
-- 
2.31.1

